test/CMakeLists.txt

```diff
@@ -235,6 +235,12 @@ target_link_libraries(ArborX_Test_BoostAdapters.exe PRIVATE ArborX Boost::unit_t
 target_compile_definitions(ArborX_Test_BoostAdapters.exe PRIVATE BOOST_TEST_DYN_LINK)
 add_test(NAME ArborX_Test_BoostAdapters COMMAND ArborX_Test_BoostAdapters.exe)
 
+add_executable(ArborX_Test_InterpDetailsSymmPInvSVD.exe tstInterpDetailsSymmPInvSVD.cpp utf_main.cpp)
```

Comment:
I will stick with `InterpDetailsSVD` because it mimics the location and name of the tested file (in `src/interpolation/details`). But maybe would it be better to have `ArborX_InterpDetailsSymmetricPseudoInverseSVD.hpp` elsewhere?